app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html

```diff
@@ -216,6 +232,20 @@ <h2 class="heading-medium">@messages(s"$key$userKey.whatNext")</h2>
 
               @warning(messages(s"$key$userKey.warning", TimeUtils.govDisplayFormat(charge.dueDate)))
 
+              @if(isAgent){
+                @if(isLateReturn){
+                  <p>@messages(s"$key.agent.penalty.warning"))</p>
+                }
+              }else if (isATrust) {
+                @if(isLateReturn){
+                    <p>@messages(s"$key.trust.penalty.warning"))</p>
+                }
+              }else{
+                @if(isLateReturn){
+                    <p>@messages(s"$key$userKey.penalty.warning"))</p>
```

Comment:
@nubz can you check if any css class needs to be applied to the penalty warning messages?